instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/rpc/RpcClientMetricsTest.java

```diff
@@ -76,6 +79,62 @@ void collectsMetrics() {
 
     assertThat(metricReader.collectAllMetrics())
         .satisfiesExactlyInAnyOrder(
+            metric ->
+                OpenTelemetryAssertions.assertThat(metric)
```

Comment:
usually we static import `OpenTelemetryAssertions.assertThat`

Reply:
Done.